### DISCUSSION TITLE
Obter o token JWT

### DISCUSSION THREAD

**Discussion by felipeiw** [viniciussanchez/RESTRequest4Delphi#103] [Q&A] [answered]
Pessoal to iniciando testes com o restrequest, e estou com dificuldade pra montar uma consulta post para obter o token jwt, tenho o token de autenticação, e na documentação diz o seguinte:
Ao chamar a ação Login, é necessário informar o Content-Type 'application/json' no Header e o login_token em um objeto JSON no Body

no postman apenas coloquei em " " o token de autenticação em body, e quando executo retorna o token jwt, mais nao to conseguindo com o restrequest, os amigos poderiam por favor me ajudar ?
obrigado

**Comment by viniciussanchez** [comment]
Manda um print de como esta fazendo em ambos os locais

**Comment by felipeiw** [comment]
var
  resp: IResponse;
begin
  resp := TRequest.New.BaseURL('https://apiv3.ideris.com.br/Login')
          .Accept('application/json')
          .AddBody('394t32k9xfkuztrjwjfeefpysbjyf6hb67a98u2ecem6xjx6bmen7gbxd2isp42tcis')
          .Post;

  if (resp.StatusCode = 0) then
    raise Exception.Create('Não foi possível acessar o servidor')
  else if (resp.StatusCode <> 200) then
    raise Exception.Create(resp.Content)
  else if (resp.StatusCode = 200) then
  begin
    Memo1.Lines.Clear;
    Memo1.Lines.Add(Format('Http status: %d', [resp.StatusCode]));
    Memo1.Lines.Add(Format('Http text: %s', [resp.StatusText]));
    Memo1.Lines.Add(Format('Response content: %s', [resp.Content]));
  end;
![Postman](https://user-images.githubusercontent.com/105029973/188196170-878c36f6-3bf2-4429-8389-8f4322d62562.png)

**Reply by viniciussanchez** [reply]
Tente assim:
```delphi
TRequest.New.BaseURL('https://apiv3.ideris.com.br/Login')
  .ContentType('application/json')
  .AddBody('"394t32k9xfkuztrjwjfeefpysbjyf6hb67a98u2ecem6xjx6bmen7gbxd2isp42tcis"')
  .Post;
```

**Comment by felipeiw** [comment]
Vinicius, deu certinho, muito obrigado !!!

**Comment by viniciussanchez** [comment, marked as answer]
```delphi
TRequest.New.BaseURL('https://apiv3.ideris.com.br/Login')
  .ContentType('application/json')
  .AddBody('"394t32k9xfkuztrjwjfeefpysbjyf6hb67a98u2ecem6xjx6bmen7gbxd2isp42tcis"')
  .Post;
```
